bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2022A.xml

```diff
@@ -0,0 +1,182 @@
+<conf>
+    <smtp>smtp.hi.gemini.edu</smtp>
+
+    <bcc>swalker@gemini.edu</bcc>
+    <bcc>cquiroz@gemini.edu</bcc>
+    <bcc>sraaphorst@gemini.edu</bcc>
+    <bcc>rnorris@gemini.edu</bcc>
+
+    <type name="FastTurnaround">
+        <dir>/home/software/proposals/fast_turnaround/proposals/2021B</dir>
```

Comment:
Should this be `2022A` instead of `2021B`?

Reply:
Yeah sorry, I switched to the next branch too quickly